src/ext/java/org/opentripplanner/ext/emissions/EmissionsService.java

```diff
@@ -0,0 +1,29 @@
+package org.opentripplanner.ext.emissions;
+
+import java.util.Optional;
+import org.opentripplanner.framework.lang.Sandbox;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+
+/**
+ * A service for getting emissions information for routes.
+ */
+@Sandbox
+public interface EmissionsService {
+  /**
+   * Get specific type of emissions per meter for a specific route.
+   * @param feedScopedRouteId
+   * @param emissionType
+   * @return Emissions per meter
+   */
+  Optional<Double> getEmissionsPerMeterForRoute(
```

Comment:
These functions now return "emissions" but the return value is just a Double. I think we should either rename these to return co2 instead of emissions or make them return some emissions object which has co2 field. I'm sort of leaning towards the latter. This same naming issue affects a lot of methods in this pr.